lib/index.js

```diff
@@ -538,8 +540,23 @@ api.verify = async ({
     });
   }
 
-  const token = record.meta['bedrock-authn-token'].tokens[type];
-  if(token.expires && new Date() >= new Date(token.expires)) {
+  let tokenValue = record.meta['bedrock-authn-token'].tokens[type];
+  const now = Date.now();
+  if(type === 'nonce') {
+    for(const token of tokenValue) {
+      if(token.sha256 === prefixedHash(hash)) {
+        if(token.expires && now >= token.expires) {
```

Comment:
`token.expires` is used in a comparison directly here.  But in `get()` and `getAll()` it's parsed by `Date()` first.  I'm not sure if these `expires` values are stored as native values or stringified dates, or how they should be stored.  Either way, seems all these APIs should handle dates the same way.

Reply:
Updated it. Did some research and found out using Date.parse is not recommended since it may cause unexpected behavior since its implementation across browsers may be different, so I have used Date() instead.